pkg/api/dashboard.go

```diff
@@ -201,6 +201,19 @@ func (hs *HTTPServer) GetDashboard(c *contextmodel.ReqContext) response.Response
 		if errors.Is(err, dashboards.ErrFolderNotFound) {
 			return response.Error(http.StatusNotFound, "Folder not found", err)
 		}
+
+		if apierrors.IsForbidden(err) {
```

Comment:
Looks like this check isn't working - when running it locally still seeing:
<img width="772" alt="Screenshot 2025-02-07 at 1 48 26 PM" src="https://github.com/user-attachments/assets/e813b1e7-2b76-4f96-b6fc-65224f4583ed" />